app/models/room.rb

```diff
@@ -1,3 +1,10 @@
 class Room < ApplicationRecord
   belongs_to :hotel
+  has_many :guest_rooms
+  has_many :guests, through: :guest_rooms
+
+  def guest_count
+    # pry
+    self.guests.count
```

Comment:
No need for the 'self.' here, can just do `guests.count`. Good job utilizing your AR associations.